Face Detection using Shape Detection API/README.md

```diff
@@ -0,0 +1,197 @@
+
+But that hasn’t stopped the people creating web technologies to move forward to have access to the hardware. Now we can access hardware using JavaScript, HTML and CSS(somewhat)!
+
+Let's start with the 1st sub-activity!
+
+---
+## Activity 3.1:- Hardware Camera Access using WebAPIs
+---
+The MediaDevices interface provides access to connected media input devices like cameras and microphones, as well as screen sharing. In essence, it lets you obtain access to any hardware source of media data.
+
+- Complete the task labelled with Activity 3.1 and head on to the next sub-activity!
+
+---
+## Activity 3.2:- Attach MediaStream to HTMLVideoElement
+---
+The `MediaStream` of the webcam must be programmatically attach to the video HTML tag to display the frames of the video on the browser.
+
+- Complete the task labelled with Activity 3.2 and head on to the next sub-activity!
+
+---
+## Activity 3.3:- Create an Instance of FaceDetector
+---
+Face detection can be done using the FaceDetector interface. FaceDetector represents the underlying platform’s component for detecting human faces in images.
+
+- Complete the task labelled with Activity 3.3 and head on to the next sub-activity!
+
+---
+## Activity 3.4:- Initiate Face Detection
+---
+
+The FaceDetector instance’s detect() method can then be used to detect human faces from the given image source.
+
+- Complete the task labelled with Activity 3.4 and come back here later!
+
+detect() methos returns a Javascript Promise which resolves to an array of DetectedFace objects for each human face that was detected from the video source. 
+
+Now, here comes the challenging task. How to draw a rectangle around the face with the co-ordinates recieved from the detector. We will see this in the next activity!
+
+---
+# Activity 4: - Display detected faces.
+---
+Bounding Box is an object describing a rectangle that indicates the position and boundaries within the image where the face was detected.
+
+Let's start with the 1st sub-activity!
+
+---
+## Activity 4.1: Draw Bounding Box for detected co-ordinates.
+---
+-  Complete the task labelled with Activity 3.4 and come back here later!
+
+The API must be able to detect faces and the boundary rectangle is drawn around the face. Congratulations!!
+
+The output should look like the Image displayed in the Introduction phase.
+
+---
+## Activity 4.2: Tinkering with Canvas Properties.
+---
+
+There are canvas properties which enabled the Canvas to draw the colorful boundary box. You can experiment with different values as per your personal preferance.
+
+- Head out to task labelled with Activity 4.2 to experiment!
+
+We're near the end of this Micro-Byte but won't conclude without challenges.
+
+> ### Micro-Challenge:- Censor the detected faces. Check the below image.
+
+![Censored Image](./images/censored.png)
+
+---
+# Mega Challenge
+---
+
+This a challenging task but definitly worth it to build!
+
+### Build a Round Robin Face Swapper.

```

Comment:
This statement makes me wanna do this MicroByte right away! Awesome addition!

Reply:
Yes. You should definitely try it out. I will share my implementation once you are done with yours.

Reply:
Sounds like a plan. :+1:

Reply:
👍🏼